src/parrotconfigs/rightparrot.js

```diff
@@ -1,13 +1,36 @@
-var parrotConfig = require("./parrot");
+var timesBigger = 9; // Very approximately accurate.
 
 var config = {
+    width:320,
+    height:229,
     flipY:true,
-    followingFrames:parrotConfig.followingFrames.map(frame => {
+
+    followingFrames:[{
+        x:16, y:11
+    },{
+        x:12, y:10
+    },{
+        x:9, y:10
+    },{
+        x:4, y:11
+    },{
+        x:4, y:12
+    },{
+        x:6, y:14
+    },{
+        x:10, y:16
+    },{
+        x:15, y:14
+    },{
+        x:17, y:13
+    },{
+        x:18, y:12
+    }].map(frame => {
```

Comment:
This seems to use the exact same frames as the `parrot` (like it originally did). I think you changed this because the size is now 9 times-or-so bigger, but we should still keep the original size somewhere for it to be reusable

Reply:
Ah yeah, sorry, I went down my own rabbit hole with further commits on my master branch. :) I've created a new PR with just the mega white border updates if you want to test that. PR #34